sdk/communication/tests.yml

```diff
@@ -16,16 +16,14 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: communication
-    Clouds: ACS_Public
     CloudConfig:
-      ACS_Public:
-        SubscriptionConfiguration: $(sub-config-communication-services-cloud-test-resources)
+      Public:
+        SubscriptionConfigurations:
+          - $(sub-config-azure-cloud-test-resources)
+          - $(sub-config-communication-services-cloud-test-resources-test)
```

Comment:
i suggest rename this to something like `sub-config-communication-services-cloud-test-resources-common`

Reply:
We need to change the keyvault and the variable name first right?